configs/sites/tier1/acorn/packages_intel.yaml

```diff
@@ -31,7 +31,10 @@
       - '@1.2.1 ~mkl'
     py-numpy:
       require::
-      - '@:1.23 ^openblas'
+      - '^[virtuals=lapack,blas] openblas'
+      - '@:1.25'
```

Comment:
Does this `any_of` add to the `@:1.25` or negate it?

Reply:
It basically makes it more restrictive for Intel 19 (technically both requirements are being applied).

Reply:
But the opposite is needed. We need it restricted to @:1.23 with any newer Intel, and @:1.24 with the older Intel; for GNU, it can be @:1.25.

Reply:
It's working with intel@2022 using py-numpy@1.25.2, and intel@19 with py-numpy@1.24.4. So I think I can either leave this line as it is or add a `when: intel@2022`.